### PR TITLE
LPD-31943

### PR DESCRIPTION
**Pull request by carolmariaabb**
https://liferay.atlassian.net/browse/LPD-31943

### CODE REVIEW

File: modules/apps/calendar/calendar-web/src/main/java/com/liferay/calendar/web/internal/info/item/provider/CalendarBookingInfoItemFieldValuesProvider.java [hunk 1]
```diff
@@ -84,6 +86,63 @@ calendarBooking, _getThemeDisplay())
...
+	 * See {@link
+	 * com.liferay.calendar.internal.notification.NotificationTemplateContextFactory#_getCalendarBookingURL(
+	 * User, long)}
+	 */
+	protected String getCalendarBookingURL(CalendarBooking calendarBooking) {
+		try {
+			Company company = _companyLocalService.getCompany(
+				calendarBooking.getCompanyId());
+
+			String portalURL = company.getPortalURL(
```
Comment (linolaoj) [inline modules/apps/calendar/calendar-web/src/main/java/com/liferay/calendar/web/internal/info/item/provider/CalendarBookingInfoItemFieldValuesProvider.java:99]: Hi @carolmariaabb, should portalURL be reordered?

**Review by linolaoj — CHANGES_REQUESTED** [review]

File: modules/apps/calendar/calendar-web/src/main/java/com/liferay/calendar/web/internal/info/item/provider/CalendarBookingInfoItemFieldValuesProvider.java (hunk 1 above)
Comment (carolmariaabb) [reply modules/apps/calendar/calendar-web/src/main/java/com/liferay/calendar/web/internal/info/item/provider/CalendarBookingInfoItemFieldValuesProvider.java]: @linolaoj I updated the code. Thanks!

**Comment by carolmariaabb** [comment]
ci:test:sf

**Review by linolaoj — APPROVED** [review]

**Comment by linolaoj** [comment]
ci:forward

**Pull request closed without merging**